datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -522,32 +522,41 @@ def tag(check, version, push, dry_run):
     else:
         checks = [check]
 
+    # Check for any new tags
+    tagged = False
+
     for check in checks:
-        echo_success('Check `{}`'.format(check))
+        echo_info('{}:'.format(check))
 
         # get the current version
         if not version:
             version = get_version_string(check)
 
         # get the tag name
         release_tag = get_release_tag_string(check, version)
-        echo_waiting('Tagging HEAD with {}'.format(release_tag))
+        echo_waiting('Tagging HEAD with {}... '.format(release_tag), indent=True, nl=False)
 
         if dry_run:
             version = None
+            click.echo()
             continue
 
         result = git_tag(release_tag, push)
 
-        # For automation we may want to cause failures for extant tags
         if result.code == 128 or 'already exists' in result.stderr:
-            echo_warning('Tag `{}` already exists, skipping...'.format(release_tag))
+            echo_warning('already exists')
         elif result.code != 0:
-            abort(code=result.code)
+            abort('\n{}{}'.format(result.stdout, result.stderr), code=result.code)
```

Comment:
do we want to abort part way through the loop?

Reply:
Yes b/c there may be some git error we don't want to ignore like bad auth, etc.